bullet/src/SDFFeatures.cc

```diff
@@ -359,6 +359,13 @@ Identity SDFFeatures::ConstructSdfJoint(
     convertVec(ignition::math::eigen3::convert(axisChild)),
     convertVec(ignition::math::eigen3::convert(axisParent)));
 
+  // Limit movement for fixed joints
+  if(type == ::sdf::JointType::FIXED) {
+    btHingeConstraint *hingeJoint = dynamic_cast<btHingeConstraint *>(joint);
```

Comment:
You don't need the dynamic cast, **joint** is a `btHingeContraint` at that point, just after retrieving it from the joints map you need the dynamic_cast

Reply:
You are right, I removed that cast with 72b656f